server/tools/eslint/config.js

```diff
@@ -4,16 +4,18 @@ module.exports = {
     'commonjs': true,
     'es2021': true,
   },
-  'extends': [
-    'google',
-  ],
+  // Extending the Google JS style guide leads to thousands of errors.
+  // If we'd like to do this it will require significant effort.
+  // 'extends': [
+  //   'google'
+  // ],
   'parserOptions': {
     'ecmaVersion': 'latest',
   },
   'rules': {
-    "indent": ["error", 4],
-    "no-redeclare": "error",
-    "max-len": ["error", { "code": 128 }],
+    // "indent": ["error", 4],
+    "no-redeclare": "warn",
```

Comment:
Why would redeclarations just be a warning?

Reply:
I should have left a comment to this effect: it would have required a decent-size overhaul as a part of this PR since we redeclare all the time (especially "var p = document.getObjectByID..."). I was thinking of leaving it was warning for now and then following up with a change to error if we decided we wanted to make this change.